tRecorderApi/api/CreateZIP.py

```diff
@@ -1,31 +0,0 @@
-# import required frameworks

```

Comment:
@justin-vanwinkle Yeah. We didn't need this. It was more of an example of how to make it work.